src/app/modules/common/services.js

```diff
@@ -535,6 +555,65 @@ angular.module('BloomLibraryApp.services', ['restangular'])
                 rBook.customPUT(putParams);
             };
 
+        this.relateBooksById = function() {
+            //Context of arguments is lost going into IIFE
+            var tArgs = arguments;
```

Comment:
Since we apparently have a variable number or arguments, an comment explaining what args are expected would be very helpful.